From: Robert Schwebel <r.schwebel@pengutronix.de>
Date: Sat, 24 Jan 2015 20:57:27 +0100
Subject: [PATCH] configure.ac: don't specify hardcoded include search path

If we crosscompile rrdtool or packetize for a distribution, libxml2
headers may be installed in a SYSROOT/DESTDIR directory, not in
/usr/include. For the cross compile usecase, this leaks a host header
file in.

Signed-off-by: Robert Schwebel <r.schwebel@pengutronix.de>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 9e934595b102..793e02874d4b 100644
--- a/configure.ac
+++ b/configure.ac
@@ -540,7 +540,7 @@ EX_CHECK_ALL(cairo,      cairo_pdf_surface_create,      cairo-pdf.h,
 EX_CHECK_ALL(cairo,      cairo_ps_surface_create,       cairo-ps.h,             cairo-ps,    1.10.2,  http://cairographics.org/releases/, "")
 EX_CHECK_ALL(pangocairo-1.0,  pango_cairo_context_set_font_options,  pango/pango.h,  pangocairo,  1.28.4,    http://ftp.gnome.org/pub/GNOME/sources/pango/1.28, "")
 fi
-EX_CHECK_ALL(xml2,       xmlParseFile,                  libxml/parser.h,        libxml-2.0,        2.7.8,  http://xmlsoft.org/downloads.html, /usr/include/libxml2)
+EX_CHECK_ALL(xml2,       xmlParseFile,                  libxml/parser.h,        libxml-2.0,        2.7.8,  http://xmlsoft.org/downloads.html)
 
 if test "$EX_CHECK_ALL_ERR" = "YES"; then
   AC_MSG_ERROR([Please fix the library issues listed above and try again.])
